projects/plugins/jetpack/extensions/blocks/conversation/example.js

```diff
@@ -0,0 +1,65 @@
+const participants = [
+	{
+		participantSlug: 'participant-0',
+		participant: 'Rosalind',
+		hasBoldStyle: true,
+		hasUppercaseStyle: true,
+	},
+	{
+		participantSlug: 'participant-1',
+		participant: 'Orlando',
+		hasItalicStyle: true,
+		hasUppercaseStyle: true,
+	},
+];
+
+const template = [
+	{
+		name: 'core/heading',
+		attributes: {
+			content: 'Shakespeare text',
+			level: 4,
+		},
+	},
+	{
+		name: 'jetpack/dialogue',
+		attributes: {
+			...participants[ 0 ],
+			content: 'O, my dear Orlando, how it grieves me to see thee wear thy heart in a scarf!',
```

Comment:
To clarify, I meant making this translatable, e.g.

```suggestion
			content: __( 'O, my dear Orlando, how it grieves me to see thee wear thy heart in a scarf!', 'jetpack' ),
```

Reply:
sorry commeted 6343cb34011ee2b8b450e2e36dc754eca63274c0 before to ready your suggestion :-(